cmd/etcd.go

```diff
@@ -113,76 +113,79 @@ func showDDLInfoCommandFunc(cmd *cobra.Command, args []string) {
 	cmd.Println(res)
 }
 
-func delOwnerCampaign(cmd *cobra.Command, args []string) {
+func delKeyCommandFunc(cmd *cobra.Command, args []string) {
 	if len(args) != 1 {
-		cmd.Println(cmd.UsageString())
+		cmd.Printf("Only one argument!")
 		return
 	}
 
-	leaseID := args[0]
+	key := args[0]
+
+	if !strings.HasPrefix(key, "/tidb/ddl/") {
```

Comment:
Could we define `/tidb/ddl/` as a constant?

Reply:
Done.
Do we need to make its scope global?
( I think in function is enough.